src/faults/index.ts

```diff
@@ -2,14 +2,19 @@ import { RPCServiceClient } from "@textile/grpc-powergate-client/dist/index/faul
 import { Config, faultsTypes } from "../types"
 import { promise } from "../util"
 
+type Faults = ReturnType<typeof bindClient>
+
 /**
  * Creates the Faults API client
  * @param config A config object that changes the behavior of the client
  * @returns The Faults API client
  */
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-export const createFaults = (config: Config) => {
+export const createFaults = (config: Config): Faults => {
```

Comment:
including Faults here now will cause a ts error on line 5, `Type alias 'Faults' circularly references itself.`, ts is smart enough to connect the two without listing it here.

Reply:
as in, the examples that used the type the way i had it checked out fine.